notebooks/book2/10/mc_estimate_pi.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>ge rid of warnings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>